### ISSUE TITLE
`ClassVar` not supported?

### ISSUE THREAD

**Issue by sneakers-the-rat** [mansenfranzen/autodoc_pydantic#123]
Hello! Wondering if i'm doing something wrong.

I have a model with `typing.ClassVar`s to indicate class, rather than instance attributes:

eg:

```python
from pydantic import BaseModel
from abc import ABC
from typing import ClassVar, Type
from PySide2 import QtWidgets

class Input(BaseModel, ABC):
    """
    Metaclass to parametrically spawn a Qt Input widget for a given type.
   """
    # class variables set by subtypes
    widget: ClassVar[Type[QtWidgets.QWidget]] = None
```

This throws the following stack:

```
  File "./autodoc_pydantic/directives/options/composites.py", line 259, in wrapped
    result = func(*args, **kwargs)
  File "./autodoc_pydantic/directives/autodocumenters.py", line 539, in add_directive_header
    self.add_default_value_or_marker()
  File "./autodoc_pydantic/directives/autodocumenters.py", line 586, in add_default_value_or_marker
    if self.needs_required_marker:
  File "./autodoc_pydantic/directives/autodocumenters.py", line 549, in needs_required_marker
    is_required = self.pydantic.inspect.fields.is_required(field_name)
  File "./autodoc_pydantic/inspection.py", line 188, in is_required
    return self.get(field_name).required
  File "./autodoc_pydantic/inspection.py", line 137, in get
    return self.attribute[name]
KeyError: 'widget'
```

So it seems like it's being seen by the fields list, but not populated into the attributes dictionary? Not sure if this is a me thing, but if not lmk how I can help with a PR :)

**Comment by mansenfranzen** [comment]
The error occurs because `autodoc_pydantic` treats `typing.ClassVar` as a pydantic field which is not correct. This is similar to #128. A fix will be available soon.

**Comment by sneakers-the-rat** [comment]
thank you! sorry i don't have any time to help out with the patch at the moment.

**Comment by mansenfranzen** [comment]
This bug is addressed via #130 .

Before merging the related PR, it would be great if you could test the bugfix on your site. To do so, please install the current dev release in your doc-building-environment via `pip install git+https://github.com/mansenfranzen/autodoc_pydantic.git@v1.7.2-a.1` and rebuild your docs.

**Comment by sneakers-the-rat** [comment]
you are too sweet :)
